packages/suite/src/views/settings/index.tsx

```diff
@@ -366,7 +364,7 @@ const Settings = () => {
                                     values={{
                                         version: (
                                             <VersionTooltip content={process.env.COMMITHASH || ''}>
-                                                <VersionLink
```

Comment:
Underline should not be there
![Screenshot 2021-09-07 at 14 35 48](https://user-images.githubusercontent.com/33235762/132345594-005d9adb-9e6b-4bd2-829d-07025d5d6c72.png)
![Screenshot 2021-09-07 at 14 35 55](https://user-images.githubusercontent.com/33235762/132345597-bbfa4fd0-7a9f-4a84-a1dc-0c17a37e3ff3.png)

Reply:
❌ @ravinalamada There is also version of app in settings where it is still underlined